src/AsyncCreatable.js

```diff
@@ -13,10 +13,6 @@ const AsyncCreatable = React.createClass({
 							<Select
 								{...asyncProps}
 								{...creatableProps}
-								onInputChange={(input) => {
-									creatableProps.onInputChange(input);
```

Comment:
Was that the purpose of calling it here, isn't createbleProps the onInputChange that the consumer passes in? Why doesn't it get called, is it being overridden by the Select component?